examples/1.initialization.py

```diff
@@ -15,7 +15,7 @@ async def main():
 
   # initialize the client
   # on boards user on firefly. Must be set to true for first time use
-  await client.init(True) 
+  await client.init(True,"") 
```

Comment:
second param is optional, so no need to provide the empty string `""` over here. You can update the comment saying `Initializing client for the private key provided. The second argument `apiToken` is optional`